src/XRP/shared/rippled-web-socket-schema.ts

```diff
@@ -0,0 +1,111 @@
+/* Schema for adding type information to Web Socket objects. */
+
+/**
+ * The standard format for a request to the JSON RPC exposed by a rippled node.
+ * @see https://xrpl.org/request-formatting.html
+ */
+interface WebSocketRequestOptions {
+  command: string
+  id: string
+  streams?: string[]
+  accounts?: string[]
```

Comment:
hmmm, where do `streams` and `accounts` come from?  looking at "https://xrpl.org/request-formatting.html" I don't see them as part of a standard websocket request.  Oh I see, they're specific to the `subscribe` request... Is there a better way to create this type by keeping `id` and `command` in their own generic request type, and then maybe creating specific request types by creating a union of the generic request with the specific fields for the request?  (just thinking about this since we'll probably be adding API options to the ws client)